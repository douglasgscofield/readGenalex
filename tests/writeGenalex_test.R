library(readGenalex)
data(exmaple_genotypes)
writeGenalex(example_genotypes, "")
writeGenalex(example_genotypes, "", quote = TRUE)
