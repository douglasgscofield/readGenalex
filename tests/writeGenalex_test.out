5	16	2	7	9
Example genotypes			pop1	pop2
id	Site	loc1		loc2		loc3		loc4		loc5	
comp1	pop1	2	3	1	1	2	2	2	3	4	2
comp2	pop1	2	3	1	3	2	2	2	1	4	3
comp3	pop1	2	3	1	1	2	2	2	3	4	2
comp4	pop1	2	3	1	3	2	2	2	3	4	3
comp5	pop1	2	3	1	1	2	2	2	3	4	2
comp6	pop1	2	3	1	1	2	2	2	3	4	2
comp7	pop1	3	3	2	3	3	1	3	3	4	3
comp8	pop2	1	1	2	1	2	5	3	3	3	1
comp9	pop2	2	2	2	1	2	5	2	2	4	3
comp10	pop2	2	3	1	1	4	4	3	3	6	1
comp11	pop2	2	1	1	1	3	2	2	3	2	3
comp12	pop2	3	3	2	1	2	2	3	1	3	3
comp13	pop2	3	3	4	4	2	5	2	3	2	3
comp14	pop2	2	1	1	1	2	2	3	1	4	5
comp15	pop2	2	1	1	1	2	4	3	1	6	7
comp16	pop2	2	1	1	1	2	2	3	1	4	4
5	16	2	7	9
"Example genotypes"			"pop1"	"pop2"
"id"	"Site"	"loc1"		"loc2"		"loc3"		"loc4"		"loc5"	
"comp1"	"pop1"	"2"	"3"	"1"	"1"	"2"	"2"	"2"	"3"	"4"	"2"
"comp2"	"pop1"	"2"	"3"	"1"	"3"	"2"	"2"	"2"	"1"	"4"	"3"
"comp3"	"pop1"	"2"	"3"	"1"	"1"	"2"	"2"	"2"	"3"	"4"	"2"
"comp4"	"pop1"	"2"	"3"	"1"	"3"	"2"	"2"	"2"	"3"	"4"	"3"
"comp5"	"pop1"	"2"	"3"	"1"	"1"	"2"	"2"	"2"	"3"	"4"	"2"
"comp6"	"pop1"	"2"	"3"	"1"	"1"	"2"	"2"	"2"	"3"	"4"	"2"
"comp7"	"pop1"	"3"	"3"	"2"	"3"	"3"	"1"	"3"	"3"	"4"	"3"
"comp8"	"pop2"	"1"	"1"	"2"	"1"	"2"	"5"	"3"	"3"	"3"	"1"
"comp9"	"pop2"	"2"	"2"	"2"	"1"	"2"	"5"	"2"	"2"	"4"	"3"
"comp10"	"pop2"	"2"	"3"	"1"	"1"	"4"	"4"	"3"	"3"	"6"	"1"
"comp11"	"pop2"	"2"	"1"	"1"	"1"	"3"	"2"	"2"	"3"	"2"	"3"
"comp12"	"pop2"	"3"	"3"	"2"	"1"	"2"	"2"	"3"	"1"	"3"	"3"
"comp13"	"pop2"	"3"	"3"	"4"	"4"	"2"	"5"	"2"	"3"	"2"	"3"
"comp14"	"pop2"	"2"	"1"	"1"	"1"	"2"	"2"	"3"	"1"	"4"	"5"
"comp15"	"pop2"	"2"	"1"	"1"	"1"	"2"	"4"	"3"	"1"	"6"	"7"
"comp16"	"pop2"	"2"	"1"	"1"	"1"	"2"	"2"	"3"	"1"	"4"	"4"
